js/background.js

```diff
@@ -38,7 +38,7 @@
         });
     });
 
-    var SERVER_URL = 'https://textsecure-service-staging.whispersystems.org';
+    var SERVER_URL = 'https://textsecure-service-ca.whispersystems.org';
```

Comment:
`-ca` is our 'certificate authority' endpoint, as opposed to the new self-signed certificate endpoint we use in the standalone app. It's what the Chrome app uses in production.

But this is a bug, because the release process already updates this file to go from `-staging` in dev mode to `-ca` in production mode: https://github.com/signalapp/Signal-Desktop/blob/636fae1857e70d56c0d9242ce4aba35aae3c0054/Gruntfile.js#L128-L130

This is what I do when I develop locally.